patch MinPath*.py script to use glpsol provided through GLPK dependency rather than included one
author: Kenneth Hoste (HPC-UGent)
--- MinPath/MinPath1.4.py.orig	2019-02-28 12:23:26.875861180 +0100
+++ MinPath/MinPath1.4.py	2019-02-28 12:22:47.142287000 +0100
@@ -19,6 +19,7 @@
 import math
 
 minpath = os.environ.get('MinPath')
+glpkroot = os.getenv("EBROOTGLPK")
 path0 = "/u/yye/Pathways/MinPath"
 if minpath or os.path.exists(path0):
         if os.path.exists(path0):
@@ -26,7 +27,7 @@
 else:
         sys.exit("Environment variable MinPath not set")
 
-keggPath0, seedPath0, mapPath0, glpsol0 = minpath + "/data", minpath + "/data", minpath + "/data", minpath + "/glpk-4.6/examples/glpsol"
+keggPath0, seedPath0, mapPath0, glpsol0 = minpath + "/data", minpath + "/data", minpath + "/data", glpkroot + "/bin/glpsol"
 
 def intmatrix(dim1, dim2):
 	mat = []
